File: public/js/src/models/entities/employee.js

// Класс "сотрудник"
export class Employee {
    id          // ID
    firstname   // Имя
    lastname    // Фамилия
    middlename  // Отчество
    date        // Дата рождения
    adress      // Адрес

    constructor(id, firstname, lastname, middlename, date, adress) {
        this.Id = id
        this.Firstname = firstname
        this.Lastname = lastname
        this.Middlename = middlename
        this.Date = date
        this.Adress = adress
    }
}

File: entities/Employee.go
package entities

import (
	"time"
)

type Employees struct {
	ID         int64     `json:"id"`         // идентификатор
	FirstName  string    `json:"firstname"`  // имя
	LastName   string    `json:"lastname"`   // фамилия
	MiddleName string    `json:"middlename"` // отчество
	Date       time.Time `json:"date"`       // дата рождения
	Adress     string    `json:"adress"`     // адрес
}
